alert-combiner-py/src/dynamo_utils.py

```diff
@@ -0,0 +1,306 @@
+import logging
+from datetime import datetime
+import time
+from src.constants import ALERTS_LOOKBACK_WINDOW_IN_HOURS
+import pandas as pd
+from src.utils import Utils 
+
+item_id_prefix = ""
+
+class DynamoUtils:
```

Comment:
seems like a lot of copy paste; can we refactor to make this a bit more modular?

Reply:
@christian-forta, I refactored the code a bit, but besides the read/write for the fp mitigation and end user attacks clusters, they all have some differences and I couldn't find a way of creating a more generic function without it being too "ugly".
If you have any ideas, I can update it.